communication/socket/create-vmci-socket.yml

```diff
@@ -0,0 +1,27 @@
+rule:
+  meta:
+    name: create VMCI socket
+    namespace: communication/socket
```

Comment:
do you think this is most related to "socket" functionality or VM interaction?

Reply:
im not too familar with this tech, but i wonder if the namespace should be more related to vm interaction, maybe like `hsot-interaction/virtual-machine/vmci` or something

Reply:
It's a good point and I think that VMCI sockets could fit in both categories/namespaces but ultimately they are just another socket type so `communication/socket` (or `communication/socket/vmci`) seems to be more adequate.

Reply:
cool, thanks for confirming!